_posts/2019-04-08-moveit-survey.md

```diff
@@ -0,0 +1,18 @@
+---
+author: davetcoleman
+comments: false
+date: 2019-04-08
+layout: post
+title: 2019 MoveIt Survey
+media_type: image
+media_link: /assets/images/closed-loop-control-system-11-3-15.gif
+description: Now that MoveIt 1.0 is officially released, we are polling the community of MoveIt users to get feedback about MoveIt.
+categories:
+- MoveIt!
+- ROS
+---
+
+Now that MoveIt 1.0 is officially released, we are polling the community of MoveIt users to get feedback about MoveIt. How easy it is to setup/use? What features are popular? What new features users would like to see in MoveIt? 
+This 20 question survey should take less than 5 minutes to fill out. This form will stay open until May 15th. 
+
+### [MoveIt Survey 2019 Link](https://docs.google.com/forms/d/e/1FAIpQLSdN4ntoe9KHpnx-q_GkDDZs5ID9wtTRKhCzzNvRMMPsC4Uj8g/viewform)
```

Comment:
This requires me to log in.

Was that intentional?

Reply:
Certainly not, fixed